ios/RNDateTimePickerManager.m

```diff
@@ -24,7 +24,7 @@ @implementation RCTConvert(UIDatePicker)
 
 @implementation RNDateTimePickerManager
 
-RCT_EXPORT_MODULE()
+RCT_EXPORT_MODULE(RNDateTimePicker)
```

Comment:
any specific reason for this change? Just wondering.

Reply:
No, I follow the docs and added this change I'll test without this and check if this still working

Reply:
Done